lib/logstash/util/accessors.rb

```diff
@@ -48,7 +48,9 @@ def strict_set(accessor, value)
 
     def del(accessor)
       target, key = lookup(accessor)
-      target.is_a?(Array) ? target.delete_at(key.to_i) : target.delete(key)
+      unless target.nil?
```

Comment:
two minor style things:

* nil is like false in ruby, so we can use ```unless target```
* We can also write here something like ```return unless target```

But as I said, just minor style stuff :-)